Reason: Fix crossbuilding GIR
Upstream: No, not upstreamable since this is very specific to the Void setup
Index: b/src/libnm-client-impl/meson.build
===================================================================
--- a/src/libnm-client-impl/meson.build
+++ b/src/libnm-client-impl/meson.build
@@ -210,6 +210,16 @@ if enable_introspection
     ld_library_path = ':' + ld_library_path
   endif
   ld_library_path = meson.current_build_dir() + ld_library_path
+  
+  if meson.is_cross_build()
+    qemu_python3_cmd = [
+      '/usr/bin/qemu-@XBPS_TARGET_QEMU_MACHINE@-static',
+      '-L', '@XBPS_CROSS_BASE@',
+      '@XBPS_CROSS_BASE@/usr/bin/python3'
+      ]
+  else
+    qemu_python3_cmd = python.path()
+  endif
 
   nm_settings_docs_xml_gir = custom_target(
     'nm-settings-docs-gir.xml',
@@ -219,7 +229,7 @@ if enable_introspection
       'env',
       'GI_TYPELIB_PATH=' + gi_typelib_path,
       'LD_LIBRARY_PATH=' + ld_library_path,
-      python.path(),
+      qemu_python3_cmd,
       join_paths(meson.source_root(), 'tools', 'generate-docs-nm-settings-docs-gir.py'),
       '--lib-path', meson.current_build_dir(),
       '--gir', '@INPUT@',
